src/app/server/StorablePeerConnection.h

```diff
@@ -47,7 +47,10 @@ class DLL_EXPORT StorablePeerConnection
     FabricIndex GetFabricIndex() { return mSession.mFabric; }
 
 private:
-    static constexpr size_t KeySize();
+    // KeySize is defined in the header so we always see its definition before
+    // its uses, which is necessary for a constexpr function to actually be
+    // treated as constexpr.
+    static constexpr size_t KeySize() { return sizeof(kStorablePeerConnectionKeyPrefix) + 2 * sizeof(uint16_t); }
```

Comment:
do we need a constexpr method? why not:

```cpp
static constexpr size_t kKeySize = sizeof(kStorablePeerConnectionKeyPrefix) + 2 * sizeof(uint16_t);
```

Reply:
`StorablePeerConnection` class is no longer being used. We should just delete the class.

Reply:
Will do as followup.